### PR TITLE
support multi scala verison when built hibench

### PR DESCRIPTION
**Pull request by zybing**

### CODE REVIEW

File: stormbench/pom.xml [hunk 1]
```diff
@@ -34,5 +34,46 @@
         </property>
       </activation>
     </profile>
+
```
Comment (manuzhang) [inline stormbench/pom.xml]: why do we need to set scala version for storm ?

Comment (zybing) [inline stormbench/pom.xml]: thanks @manuzhang  In the storm/streaming pom file, there is a dependency(org.apache.kafka) which need to be specified the scala version. So I set the scala version for the storm.

Comment (manuzhang) [inline stormbench/pom.xml]: shouldn't that be the same as kafka version in `common/pom.xml`?

**Comment by carsonwang** [comment]
@zybing, this is a very useful addition. Here are two minor comments:
Can you also update the build document in part `Specify Spark Version`? Add `scala=2.11` to the example as it is for spark 2.0. With this change, a user has to specify scala=2.11 and spark=2.0 if he wants to build for the Default spark 2.0 release. Ideally, we should use scala 2.11 for spark 2.0 even if the user doesn't specify the scala version.
Can you also add this scala option in Travis for testing?

**Comment by zybing** [comment]
@carsonwang ,I've speicfied ths spark2.0(1.6) with scala2.11(2.10) by default.  modifying the travis part, and updating the build-bench.md.

**Comment by carsonwang** [comment]
What if I build with `mvn -Dspark=2.0 package`? The spark module is built with scala 2.11 but the common module is built with scala 2.10 by default? This could be a problem.

**Comment by zybing** [comment]
@carsonwang I've specify the default spark version to 2.0, the default scala version to 2.11. and update build-bench.md travis.yml and dockerfile.

**Comment by carsonwang** [comment]
Looks good.

**Pull request merged**